docs/src/introduction/faq.md

```diff
@@ -0,0 +1,70 @@
+# FAQ
+
+## Where can I get help?
+
+For usage questions, please contact us via the
+[Julia Discourse](https://discourse.julialang.org/c/domain/opt). If you're
+running into bugs or have feature requests, please use the [GitHub Issue
+Tracker](https://github.com/jump-dev/Convex.jl/issues).
+
+## How does Convex.jl differ from JuMP?
+
+Convex.jl and JuMP are both modelling languages for mathematical programming
+embedded in Julia, and both interface with solvers via MathOptInterface, so many
+of the same solvers are available in both.
+
+Convex.jl converts problems to a standard conic form. This approach requires
+(and certifies) that the problem is convex and DCP compliant, and guarantees
+global optimality of the resulting solution (if the solver succeeds. For some
+models, the solver may experience numerical difficulty).
```

Comment:
this is definitely better, but note both solutions said they were "OPTIMAL". I'm not really sure it's just a numeric issue as much as an algorithmic issue; e.g. https://optimization-online.org/wp-content/uploads/2018/07/6695.pdf shows SDPA-GMP failing on some SDP's with a duality gap. Although I've seen SDPA-GMP fail easily on many problems without sufficient presolve so maybe that is a factor there too.

Reply:
Thanks for the citing my blog! I second this comment: I had in fact a similar issue with MOSEK: the problem was not strictly feasible, and as expected MOSEK failed to solve it, but then falsely claimed that it was "OPTIMAL". I submitted a [bug report](https://groups.google.com/g/mosek/c/mkYUzjmJY5U/m/vAFs_IUPCgAJ) to MOSEK but only managed to get insulted by the CEO.

Reply:
As for the proof, there's a simpler argument: a matrix `A` is positive semidefinite iff `A = B'*B` for some matrix `B`. Now if `A` has a diagonal element equal to zero, then `v'*A*v = 0` for some basis vector `v`. But  `v'*A*v = v'*B'*B*v = norm(B*v)^2`, so `B*v = 0`, which implies `B'*B*v = A*v = 0`, so the entire column must be zero.

Reply:
ah, that is a nice argument. It may be worth adding that to your blog post, or at least noting that the entire column is zero (and therefore X13 = 0) rather than jumping to X13=0, since providing the intermediate step can help readers figure it out (or figure out what to look up).

Reply:
Fair enough, I added it to the blog post (as a footnote).

Reply:
I don't want to derail this PR with a discussion, so I've added a TODO to https://github.com/jump-dev/Convex.jl/issues/344. Once this PR is merged I'm happy to review other changes.

Reply:
You could also argue that "if the solver succeeds" is true, and that solvers reporting `OPTIMAL` are a bug. That's their problem, not ours.